app/controllers/purchases_controller.rb

```diff
@@ -17,7 +17,7 @@ def index
     # Using the @purchases allows drilling down instead of always starting with the total dataset
     @purchases_quantity = @purchases.collect(&:total_quantity).sum
     @paginated_purchases_quantity = @paginated_purchases.collect(&:total_quantity).sum
-    @total_value_all_purchases = total_value(@purchases)
+    @total_value_all_purchases = @purchases.collect(&:amount_spent_in_cents).sum
```

Comment:
I think this can be just `@purchase.sum(&:amount_spent_in_cents)` as sum is smart about nil